stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/listener/internal/DeleteMessageListenerDatabase.kt

```diff
@@ -44,27 +43,7 @@ internal class DeleteMessageListenerImpl(
      * @param messageId The message id to be deleted.
      */
     override suspend fun onMessageDeletePrecondition(messageId: String): Result<Unit> {
-        return messageRepository.selectMessage(messageId)?.let { message ->
-
-            val isModerationFailed = message.user.id == clientState.user.value?.id &&
-                message.syncStatus == SyncStatus.FAILED_PERMANENTLY &&
-                message.syncDescription?.type == MessageSyncType.FAILED_MODERATION
-
-            val (channelType, channelId) = message.cid.cidToTypeAndId()
-            val channelLogic = logic.channel(channelType, channelId)
-
-            if (isModerationFailed) {
-                channelLogic.deleteMessage(message)
-                messageRepository.deleteChannelMessage(message)
-                Result.error(
-                    MessageModerationDeletedException(
-                        "Message with failed moderation has been deleted locally: $messageId"
-                    )
-                )
-            } else {
-                Result.success(Unit)
-            }
-        } ?: Result.error(ChatError(message = "No message found with id: $messageId"))
+        return Result.success(Unit)
```

Comment:
Hey @leandroBorgesFerreira,

The PR looks good, however, here we have to prevent a real `DeleteMsgRequest` from being fired to BE, cause the message with the status `FAILED_MODERATION` exists locally only.